install_config/install/advanced_install.adoc

```diff
@@ -410,16 +430,189 @@ node2.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
 To use this example, modify the file to match your environment and
 specifications, and save it as *_/etc/ansible/hosts_*.
 
-[[multi-master-multi-etcd-multi-node]]
+[[multiple-masters]]
+
+=== Multiple Masters
+
+You can configure an environment with multiple masters, multiple *etcd* hosts,
+and multiple nodes. Configuring multiple masters for high availability (HA)
+ensures that the cluster has no single point of failure.
+
+When configuring multiple masters, the advanced installation supports two high
+availability (HA) methods:
+
+.HA Master Methods
+[cols="1,5"]
+|===
+
+|`native`
+|Leverages the native HA master capabilities built into OpenShift and can be
+combined with any load balancing solution. If a host is defined in the *[lb]*
+section of the inventory file, Ansible installs and configures HAProxy
+automatically as the load balancing solution. If no host is defined, it is
```

Comment:
Wondering if we should make this super clear in terms of what needs to be load balanced with something like: "If no host is defined, it is assumed you have pre-configured a load balancer to balance the master api (port 8443) on all master hosts."